geometry_msgs/Pose gps_pose
---
geometry_msgs/Pose map_pose
